### DISCUSSION TITLE
A calendar desklet that displays appointment, but pulled from gnome-calendar

### DISCUSSION THREAD

**Discussion by starleaf1** [linuxmint/discussions#319] [Ideas]
I'd like to see a calendar desklet which displays appointments, but not just from Google like googleCalendar@javahelps.com

I'd like it to be like the Calendar panel applet (calendar@cinnamon.org), but in desklet form.
It'd be more useful for people who have Microsoft, Exchange, or other services already setup in their Online Accounts. It would also relieve the unease for people who doesn't want to authorize yet another thing to access their Google account.

**Comment by mtwebster** [comment]
How would it be more useful for those specific people?

The way the applet gets its data is the same as what evolution and gnome-calendar use (and is pretty much the standard).  There's a [Thunderbird plugin](https://addons.thunderbird.net/en-us/thunderbird/addon/eds-calendar-integration/) that will allow any calendars there to be included, and any calendars in Evolution will be visible also.

There aren't any plans to try and implement additional methods to link to calendars, the way we're doing it is 'the way'.

You can check here:
https://gitlab.gnome.org/GNOME/gnome-online-accounts

They could add calendar sharing eventually for microsoft accounts

**Reply by starleaf1** [reply]
Exactly.
There's already a calendar applet that shows events from Online Accounts, so why isn't there a desklet also?

There's currently only `googleCalendar`, which only displays Google Calendar and pulls events from Google directly. It'd be nice to have a _desklet_ that sources its events the same way the Calendar applet does.
